cli.sh

```diff
@@ -399,13 +410,18 @@ git hooks trust [forget]
     if [ -z "$1" ]; then
         mkdir -p .githooks &&
             touch .githooks/trust-all &&
-            git config githooks.trust.all Y &&
-            echo "The current repository is now trusted." &&
-            echo "  Do not forget to commit and push the trust marker!" &&
-            return
-
-        echo "! Failed to mark the current repository as trusted" >&2
-        exit 1
+            git config githooks.trust.all Y
+        if ! mkdir -p .githooks ||
```

Comment:
Why is this change necessary?
We seem to be repeating what's already been done.